DancingGoat/Utils/ProjectUtils.cs

```diff
@@ -1,11 +1,24 @@
 ﻿using System;
+using System.Configuration;
+using System.Linq;
+using System.Web;
 
 namespace DancingGoat.Utils
 {
     public class ProjectUtils
     {
+        private static string AppSettingPreviewApiKey = "PreviewApiKey";
+        private static string PreviewApiKeyCookieName = "PreviewApiKey";
+
         public static string GetProjectId()
         {
+            // check web.config first
+            if (Guid.TryParse(ConfigurationManager.AppSettings["ProjectId"], out Guid projectId))

```

Comment:
Why was this necessary?

Reply:
To ensure consistent behavior (see isPreview flag in if statement when rendering Edit mode panel).